salt/haproxy/config/haproxy.cfg.jinja

```diff
@@ -87,9 +87,10 @@ listen tls:
     bind :20007 ssl alpn h2,http/1.1 crt www.pycon.org.pem
     bind :20008 ssl alpn h2,http/1.1 crt jython.org.pem
     bind :20009 ssl alpn h2,http/1.1 crt pypy.org.pem
-    bind 0.0.0.0:443 ssl alpn h2,http/1.1 crt star.python.org.pem crt star.pypa.io.pem crt star.pyfound.org.pem crt speed.pypy.org.pem crt www.pycon.org.pem crt jython.org.pem crt pypy.org.pem crt salt-public.psf.io.pem crt planetpython.org.pem
-    bind :::443 ssl alpn h2,http/1.1 crt star.python.org.pem crt star.pypa.io.pem crt star.pyfound.org.pem  crt speed.pypy.org.pem crt www.pycon.org.pem crt jython.org.pem crt pypy.org.pem crt salt-public.psf.io.pem crt planetpython.org.pem
-    bind :20010 ssl alpn h2,http/1.1 crt star.python.org.pem crt star.pypa.io.pem crt star.pyfound.org.pem crt speed.pypy.org.pem crt www.pycon.org.pem crt jython.org.pem crt pypy.org.pem crt salt-public.psf.io.pem crt planetpython.org.pem
+    bind 0.0.0.0:443 ssl alpn h2,http/1.1 crt star.python.org.pem crt star.pypa.io.pem crt star.pyfound.org.pem crt speed.pypy.org.pem crt www.pycon.org.pem crt jython.org.pem crt pypy.org.pem crt salt-public.psf.io.pem crt planetpython.org.pem crt bugs.python.org.pem crt bugs.jython.org.pem crt issues.roundup-tracker.org.pem crt mail.roundup-tracker.org.pem
+    bind :::443 ssl alpn h2,http/1.1 crt star.python.org.pem crt star.pypa.io.pem crt star.pyfound.org.pem  crt speed.pypy.org.pem crt www.pycon.org.pem crt jython.org.pem crt pypy.org.pem crt salt-public.psf.io.pem crt planetpython.org.pem crt bugs.python.org.pem crt bugs.jython.org.pem crt issues.roundup-tracker.org.pem crt mail.roundup-tracker.org.pem
+    bind :20010 ssl alpn h2,http/1.1 crt star.python.org.pem crt star.pypa.io.pem crt star.pyfound.org.pem crt speed.pypy.org.pem crt www.pycon.org.pem crt jython.org.pem crt pypy.org.pem crt salt-public.psf.io.pem crt planetpython.org.pem crt bugs.python.org.pem crt bugs.jython.org.pem crt issues.roundup-tracker.org.pem crt mail.roundup-tracker.org.pem
```

Comment:
for these and all lines above, we'll condense the necessary certs all into one by just adding [SANs](https://en.wikipedia.org/wiki/Subject_Alternative_Name#:~:text=Subject%20Alternative%20Name%20(SAN)%20is,Subject%20Alternative%20Names%20(SANs).), so we just need to add `crt bugs.python.org`.

Reply:
do we need to proxy smtp here?

Reply:
SANS set in https://github.com/python/psf-salt/blob/3a4c7693bb551a3180071b4517d57a7fd9c7a741/pillar/base/bugs.sls#L2-L6 right?

Reply:
yep!

Reply:
> do we need to proxy smtp here?

We would set the proxying up using a configuration like [these](https://github.com/python/psf-salt/blob/448b8a7c0007bf3ffaf7098b3363956401e82929/pillar/base/haproxy.sls#L140-L157). The "listens" directive effectively wires up a TCP proxy from haproxy to the backends.